src/components/aria-live/aria-live.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  Component,
+  ElementRef,
+  AfterContentInit
+} from 'angular2/core';
+
+import {DOM} from '../../core/platform/dom/dom_adapter';
+
+@Component({
+  selector: 'md-aria-live',
+  templateUrl: './components/aria-live/aria-live.html',
+  styleUrls: ['./components/aria-live/aria-live.css'],
+})
+export class MdAriaLive implements AfterContentInit {
+
+  private announcerEl: HTMLElement;
+
+  constructor(private elementRef: ElementRef) { }
+
+  ngAfterContentInit() {
+    this.announcerEl = DOM.querySelector(this.elementRef.nativeElement, '._md-aria-announcer');
+  }
+
+  announce(message: string): void {
+    this.announcerEl.textContent = '';
```

Comment:
I tested that out, but when we we use a bound text property, then we have to set the property always to undefined, to clear the text content, setting to `''` isn't working for me in this case. (Voiceover iOS)

Additionally, I think, that a change detection is here not necessary. 
If you have other thoughts let me know, and I'll change it to Angular's Property Bound.